packages/oauth/oauth_common.js

```diff
@@ -6,6 +6,8 @@ OAuth._redirectUri = function (serviceName, config, params, absoluteUrlOptions)
   // detect whether we need to be backwards compatible by checking for
   // the absence of the `loginStyle` field, which wasn't used in the
   // code which had the "?close" argument.
+  // This logic is duplicated in the tool so that the tool can do OAuth
```

Comment:
@estark37 and I added a hack in https://github.com/meteor/meteor/commit/95e25f895949aba29abfb0a3adf10e5f124597ac to fix a problem in `meteor login` when it talks to older apps like build.meteor.com